### ISSUE TITLE
Could not play audio with src /static-f03a0c550524/audio.wav: [object MediaError]

### ISSUE THREAD

**Issue by pooooriya** [remotion-dev/remotion#1138]
hi 
i am new user
i cant use audio tag in my code its get me an error 
i didnt install ffmpeg cause i didnt need to render my video 
please help me here is my code
```
      <>
        <AbsoluteFill className="items-center justify-center" >
            <Video
                width={width}
                height={height}
                startFrom={0}
                src={video}
            />
          
            <Sequence from={10} durationInFrames={120}>
                <h1 className='text-9xl w-[100%] text-center mt-32 font-extrabold'
                 style={{opacity:interpolated}}>{text}</h1>
            </Sequence>
         </AbsoluteFill>
          <Audio src={staticFile("audio.wav")}/>
          </>

```

**Comment by pooooriya** [comment]
pervious code isnt compatible with my description cause i use CTRL+Z
yes i can able to use it in my chrome and my local machin player

**Comment by JonnyBurger** [comment]
@pooooriya Alright! What happens if you open `http://localhost:3000/static-f03a0c550524/audio.wav` in your browser?

Maybe the `audio.wav` is not inside the `public/` folder.

**Comment by pooooriya** [comment]
![image](https://user-images.githubusercontent.com/65160744/179530104-8d63cf47-4d3e-4988-af05-3d3f66392e99.png)
i got this issues in my console

**Comment by pooooriya** [comment]
its get me 404 error 
[Captive 60s.zip](https://github.com/remotion-dev/remotion/files/9132609/Captive.60s.zip)

**Comment by pooooriya** [comment]
this error also occur when i use absolute url from intenet !

**Comment by JonnyBurger** [comment]
Audio works for me if I put it in public and import it via audio tag.
It should not return 404, therefore please check the following:

Which error message do you get alongside 404? `The static/ prefix has been changed, this URL is no longer valid.` or `/Users/jonathanburger/remotion/packages/example/public/audio.wav does not exist`?
  -> Is this true? Does it not exist on the filesystem?
  -> Do you have a space or invisible character in the filename or the folder name?
  -> Do you use Chromium instead if Chrome? (To diagnose a possible codec error)

**Comment by pooooriya** [comment]
![image](https://user-images.githubusercontent.com/65160744/179532598-ed503a6c-fad4-4f05-a06f-6a055ff1da56.png)

I edit my previous comment and restart my project and check new URL as you said and it's just starting to download my audio and not play it the URL is valid because IDM shows me a file with a 5mg volume 
for your other questions
i use chrome 
I change my audio name to "audio" to avoid space or any warmful chars
and the file is existed now and i can play it easily with my system

**Comment by pooooriya** [comment]
it doesnt

**Comment by pooooriya** [comment]
after removing IDM (internet download manager) it just work now ! wtf !! :///
i kill (idm agent for browser) and allthing goes well after that !
Thank you for answering me so quickly and soberly, a package of this magnitude will definitely not be made unless the contrbiuters are patient and insightful.

**Comment by pooooriya** [comment]
is there a good idea to add this part to remotion document for audio tag ?
the cool thing was i test this in incognito !!!! and this bug occur there too !!
you cant add section or danger to remove any download app from their browser or kill them to play audio correctly 
that was only for audio . video deesnt have any issues !
thanks man again

**Comment by JonnyBurger** [comment]
Yes, it's a good idea to create a help page for this error. I'll keep it open

**Comment by JonnyBurger** [comment]
And thanks for the nice words! Indeed it takes patience but it's very good that we track these things and learn more things about what can go wrong, for example Internet Download Manager! We can create docs and help out other people who have the problem in the future! 🙌

**Comment by AliGharian** [comment]
I have the same issue

**Comment by JonnyBurger** [comment]
@AliGharian did you follow my attempts to help in this issue and can you provide info just like @pooooriya did? what can you find out using immer info in this thread?

**Comment by JonnyBurger** [comment]
#1192 will add more docs and help links for these types of problems.
